puble.css

```
@@ -0,0 +1,117 @@
+.container {
+  width: 564px;
+
+  display: flex;
+  flex-direction: column;
+  border: solid 1px #9a9a9a;
+  color: #9a9a9a;
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen,
+    Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serifZ;
+}
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 45px 16px 16px 16px;
+  border-bottom: solid 1px #c4c4c4;
+}
+.shopping-img {
+  width: 32px;
+  height: 32px;
+  cursor: pointer;
+}
+.hamburger-img {
+  width: 28.5px;
+  height: 18px;
+  cursor: pointer;
+}
+.icon {
+  width: 60px;
+  height: 60px;
+  color: white;
+  background-color: #c4c4c4;
+  border-radius: 50px;
+
+  margin: 24px 24px 0px 482px;
+  text-align: center;
+  border: none;
+  cursor: pointer;
+}
+.icon:hover {
+  transform: scale(1.1);
+  color: red;
+}
+.icon-heart {
+  cursor: pointer;
+  font-size: 24px;
+}
+.icon-number {
+  font-size: 14px;
+}
+.text {
+  text-align: center;
+  border-bottom: solid 1px #c4c4c4;
+}
+.제목 {
```

코드에서 한글을 사용해서 작동이 되더라도 지양하는 것이 좋습니다.
```suggestion
.title {
```